.env

```diff
@@ -89,6 +89,9 @@ METRIC_PARTITIONING_ROUTINE_DELAY_FIRST_RUN_SECONDS=10
 METRIC_PARTITIONINING_PREFILL_PERIOD_DAYS=7
 # Used by `ready` script to ensure metric partitions have been created.
 MINIMUM_REQUIRED_PARTITIONS=30
+REDIS_ENDPOINT_URL=redis:6379
+REDIS_PASSWORD=
+CHAIN_ID=local-chain
```

Comment:
please add comments like for other variables in this file to make what is obvious today equally obvious to future developers

Reply:
done